### ISSUE TITLE
Why duplicate login credentials in values?

### ISSUE THREAD

**Issue by xeruf** [zammad/zammad-helm#250]
In the values.yaml you have `zammadConfig.postgresql.pass` and `postgresql.auth.password`, stating explicitly that they need to be the same - why not just default one to the other so filling in one is sufficient? Same for redis, but interestingly not for minio it seems.

**Comment by monotek** [comment]
Because on of these values confiures the zammad chart and the other the postgres sub chart.

In the end these are just the default values so that the zammad chart can be easily deployed.

In production, where your config likely resides in a git repo, secrets should never be a part of it.
Thats the reason you can define exisiting secrets.

**Comment by xeruf** [comment]
yeah mine are also deployed via gitops, still it adds opportunity to screw up in the config - and I know many other charts where you only need to add the password once so there is apparently some way to access the values of the subchart

**Comment by xeruf** [comment]
.

**Comment by xeruf** [comment]
this should be a fairly quick fix but I don't have the proper experience here
